### DISCUSSION TITLE
Why can't I use my own wallet plugin normally?

### DISCUSSION THREAD

**Discussion by ClarkPgOne** [rainbow-me/rainbowkit#2079] [Feedback]
I used my own wallet plugin, and then clicked on wallet to jump to metamask, I don't know why this is happening, my wallet already supports EIP6963.Through EIP6963 test everything is normal, the making of address is the https://github.com/MetaMask/vite-react-ts-eip-6963.git.At present, I am testing locally and have not sent it online, and the online version does not support EIP6963.
![898d507fafd3bced5299d57a05f5a2c7](https://github.com/rainbow-me/rainbowkit/assets/48192529/7a9e711e-3666-4124-808d-d5a09864f104)
![b23b9cfbe9b390318c820283444f42a1](https://github.com/rainbow-me/rainbowkit/assets/48192529/8a8db88c-ecef-4362-bc32-80b9ebad1d64)

**Comment by 0xmkh** [comment]
Linking the issue https://github.com/rainbow-me/rainbowkit/issues/2080
